app/js/translateApp.module.js

```diff
@@ -0,0 +1,7 @@
+angular.module('translateApp', ['pascalprecht.translate'])
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["angular" is not defined.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2321552677/issues/source?bid=3196969&fileBranchId=3199483#l1)